.github/workflows/build.yml

```diff
@@ -103,12 +91,12 @@ jobs:
     - name: CPP Formatting Checks
```

Comment:
While you're here can you rename this step "CPP Formatting Checks (clang-format-15)" so it's super clear what version we use?